{"uid":"df79dc781877ffbf","name":"User is not allowed to enter invalid phone number","fullName":"tests.test_login.TestLogin#test_invalid_phone_number","historyId":"ce62b6e95f8e8da009fe6f309ba4fc9a","time":{"start":1683292046099,"stop":1683292046099,"duration":0},"description":"User is not allowed to enter invalid phone number","descriptionHtml":"<p>User is not allowed to enter invalid phone number</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556643b44fe3 <unknown>\n#1 0x556643883d36 <unknown>\n#2 0x5566438c0c4d <unknown>\n#3 0x5566438c0d61 <unknown>\n#4 0x5566438fc6a4 <unknown>\n#5 0x5566438e18ed <unknown>\n#6 0x5566438fa232 <unknown>\n#7 0x5566438e1693 <unknown>\n#8 0x5566438b403a <unknown>\n#9 0x5566438b517e <unknown>\n#10 0x556643b06dbd <unknown>\n#11 0x556643b0ac6c <unknown>\n#12 0x556643b144b0 <unknown>\n#13 0x556643b0bd63 <unknown>\n#14 0x556643adec35 <unknown>\n#15 0x556643b2f138 <unknown>\n#16 0x556643b2f2c7 <unknown>\n#17 0x556643b3d093 <unknown>\n#18 0x7f0fafb94b43 <unknown>","statusTrace":"request = <SubRequest 'setup' for <Function test_invalid_phone_number>>\n\n    @pytest.fixture(scope=\"function\")\n    def setup(request:pytest.FixtureRequest):\n        browser=request.config.getoption(\"--browser\")\n        if(browser in(\"chrome\",\"chrome_headless\")):\n            chrome_options = webdriver.ChromeOptions()\n            chrome_options.set_capability(\n                    \"goog:loggingPrefs\", {\"performance\": \"ALL\", \"browser\": \"ALL\"}\n                )\n        match browser:\n            case \"chrome\":\n                driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()),options=chrome_options)\n            case \"chrome_headless\":\n                chrome_options.add_argument(\"headless=new\")\n                driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()),options=chrome_options)\n            case \"firefox\":\n                driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))\n        driver.get(\"https://sporty.com/news/latest\")\n        # make sure the page is loaded\n        wait= WebDriverWait(driver,30)\n>       wait.until(expected_conditions.visibility_of_element_located(((By.XPATH,\"//*[contains(text(),'Customise your')]\"))))\n\ntests/conftest.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"f652ef594904c81c6d379f42d037ce71\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f5f851d1c60>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x556643b44fe3 <unknown>\nE       #1 0x556643883d36 <unknown>\nE       #2 0x5566438c0c4d <unknown>\nE       #3 0x5566438c0d61 <unknown>\nE       #4 0x5566438fc6a4 <unknown>\nE       #5 0x5566438e18ed <unknown>\nE       #6 0x5566438fa232 <unknown>\nE       #7 0x5566438e1693 <unknown>\nE       #8 0x5566438b403a <unknown>\nE       #9 0x5566438b517e <unknown>\nE       #10 0x556643b06dbd <unknown>\nE       #11 0x556643b0ac6c <unknown>\nE       #12 0x556643b144b0 <unknown>\nE       #13 0x556643b0bd63 <unknown>\nE       #14 0x556643adec35 <unknown>\nE       #15 0x556643b2f138 <unknown>\nE       #16 0x556643b2f2c7 <unknown>\nE       #17 0x556643b3d093 <unknown>\nE       #18 0x7f0fafb94b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup","time":{"start":1683292046100,"stop":1683292089701,"duration":43601},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556643b44fe3 <unknown>\n#1 0x556643883d36 <unknown>\n#2 0x5566438c0c4d <unknown>\n#3 0x5566438c0d61 <unknown>\n#4 0x5566438fc6a4 <unknown>\n#5 0x5566438e18ed <unknown>\n#6 0x5566438fa232 <unknown>\n#7 0x5566438e1693 <unknown>\n#8 0x5566438b403a <unknown>\n#9 0x5566438b517e <unknown>\n#10 0x556643b06dbd <unknown>\n#11 0x556643b0ac6c <unknown>\n#12 0x556643b144b0 <unknown>\n#13 0x556643b0bd63 <unknown>\n#14 0x556643adec35 <unknown>\n#15 0x556643b2f138 <unknown>\n#16 0x556643b2f2c7 <unknown>\n#17 0x556643b3d093 <unknown>\n#18 0x7f0fafb94b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/selenium-python/selenium-python/tests/conftest.py\", line 42, in setup\n    wait.until(expected_conditions.visibility_of_element_located(((By.XPATH,\"//*[contains(text(),'Customise your')]\"))))\n  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},{"name":"attach_artifacts_on_failure","time":{"start":1683292046100,"stop":1683292046100,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"User is not allowed to enter invalid phone number","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x556643b44fe3 <unknown>\n#1 0x556643883d36 <unknown>\n#2 0x5566438c0c4d <unknown>\n#3 0x5566438c0d61 <unknown>\n#4 0x5566438fc6a4 <unknown>\n#5 0x5566438e18ed <unknown>\n#6 0x5566438fa232 <unknown>\n#7 0x5566438e1693 <unknown>\n#8 0x5566438b403a <unknown>\n#9 0x5566438b517e <unknown>\n#10 0x556643b06dbd <unknown>\n#11 0x556643b0ac6c <unknown>\n#12 0x556643b144b0 <unknown>\n#13 0x556643b0bd63 <unknown>\n#14 0x556643adec35 <unknown>\n#15 0x556643b2f138 <unknown>\n#16 0x556643b2f2c7 <unknown>\n#17 0x556643b3d093 <unknown>\n#18 0x7f0fafb94b43 <unknown>","statusTrace":"request = <SubRequest 'setup' for <Function test_invalid_phone_number>>\n\n    @pytest.fixture(scope=\"function\")\n    def setup(request:pytest.FixtureRequest):\n        browser=request.config.getoption(\"--browser\")\n        if(browser in(\"chrome\",\"chrome_headless\")):\n            chrome_options = webdriver.ChromeOptions()\n            chrome_options.set_capability(\n                    \"goog:loggingPrefs\", {\"performance\": \"ALL\", \"browser\": \"ALL\"}\n                )\n        match browser:\n            case \"chrome\":\n                driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()),options=chrome_options)\n            case \"chrome_headless\":\n                chrome_options.add_argument(\"headless=new\")\n                driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()),options=chrome_options)\n            case \"firefox\":\n                driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))\n        driver.get(\"https://sporty.com/news/latest\")\n        # make sure the page is loaded\n        wait= WebDriverWait(driver,30)\n>       wait.until(expected_conditions.visibility_of_element_located(((By.XPATH,\"//*[contains(text(),'Customise your')]\"))))\n\ntests/conftest.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"f652ef594904c81c6d379f42d037ce71\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f5f851d1c60>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x556643b44fe3 <unknown>\nE       #1 0x556643883d36 <unknown>\nE       #2 0x5566438c0c4d <unknown>\nE       #3 0x5566438c0d61 <unknown>\nE       #4 0x5566438fc6a4 <unknown>\nE       #5 0x5566438e18ed <unknown>\nE       #6 0x5566438fa232 <unknown>\nE       #7 0x5566438e1693 <unknown>\nE       #8 0x5566438b403a <unknown>\nE       #9 0x5566438b517e <unknown>\nE       #10 0x556643b06dbd <unknown>\nE       #11 0x556643b0ac6c <unknown>\nE       #12 0x556643b144b0 <unknown>\nE       #13 0x556643b0bd63 <unknown>\nE       #14 0x556643adec35 <unknown>\nE       #15 0x556643b2f138 <unknown>\nE       #16 0x556643b2f2c7 <unknown>\nE       #17 0x556643b3d093 <unknown>\nE       #18 0x7f0fafb94b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[],"attachments":[{"uid":"b2189d1d4236b179","name":"stderr","source":"b2189d1d4236b179.txt","type":"text/plain","size":323}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"attach_artifacts_on_failure::0","time":{"start":1683292089753,"stop":1683292089754,"duration":1},"status":"broken","statusMessage":"AttributeError: type object 'TestLogin' has no attribute 'driver'\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 918, in _teardown_yield_fixture\n    next(it)\n  File \"/home/runner/work/selenium-python/selenium-python/tests/conftest.py\", line 55, in attach_artifacts_on_failure\n    \"Browser\": request.cls.driver.name,\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"labels":[{"name":"epic","value":"Login functionality"},{"name":"story","value":"Login functionality"},{"name":"tag","value":"@pytest.mark.usefixtures('setup')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az442-882"},{"name":"thread","value":"2180-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["@pytest.mark.usefixtures('setup')"]},"source":"df79dc781877ffbf.json","parameterValues":[]}